docs/topics/db/models.txt

```diff
@@ -58,6 +60,15 @@ Some technical notes:
   syntax, but it's worth noting Django uses SQL tailored to the database
   backend specified in your :doc:`settings file </topics/settings>`.
 
+* The foreign key relationships are made explicit by a ``FOREIGN KEY``
+  constraint.
+
+* It's tailored to the database you're using, so database-specific field types
+  such as ``auto_increment`` (MySQL), ``bigint PRIMARY KEY GENERATED BY DEFAULT
+  AS IDENTITY`` (PostgreSQL), or ``integer primary key autoincrement`` (SQLite)
+  are handled for you automatically. Same goes for the quoting of field names
```

Comment:
"The same goes"

Reply:
@evildmp addressed in 6a55b66facfc2d421c796c7c1e1609d67c506df0